docs/content/FormControl.mdx

```diff
@@ -32,6 +32,88 @@ import {MarkGithubIcon} from '@primer/octicons-react'
 </FormControl>
 ```
 
+### With a custom input
```

Comment:
Nit about ordering: This section should come after "With complex inputs", the order in which we show examples should be from "most recommended" to "least recommended"

also Nit: This example talks about caption and validation before it is introduced in the "With a caption" much later. Maybe we should move it further down?